dataRetriever/errors.go

```diff
@@ -7,8 +7,8 @@ import (
 // ErrNilMessage signals that a nil message has been received
 var ErrNilMessage = errors.New("nil message")
 
-// ErrNoConnectedPeerToSendRequest signals that the connected peers list is empty and can not send request
-var ErrNoConnectedPeerToSendRequest = errors.New("connected peers list is empty. Can not send request")
+// ErrSendRequest signals that the connected peers list is empty or errors appeared when sending requests
+var ErrSendRequest = errors.New("can not send request, peer list empty or send error")
```

Comment:
maybe refactor to: "cannot send request: peer list is empty or errors during the sending"

Reply:
changed